src/frame.js

```diff
@@ -46,7 +46,7 @@ class Frame {
 
         // Parse headers in reverse order so that for repeated headers, the 1st
         // value is used
-        for (let line of headerLines.reverse()) {
+        for (let line in headerLines.reverse()) {
```

Comment:
You actually miss the differences. TLDR; for of is safer & more powerful https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in